packages/app/app/index.js

```diff
@@ -7,22 +8,26 @@ import { AppContainer } from 'react-hot-loader';
 import { I18nextProvider } from 'react-i18next';
 import logger from 'electron-timber';
 import Img from 'react-image-smooth-loading';
+import * as Sentry from '@sentry/electron';
 
 import artPlaceholder from '../resources/media/art_placeholder.png';
 import i18n, { setupI18n } from './i18n';
 import App from './App';
 import configureStore from './store/configureStore';
 
-const store = configureStore();
-logger.hookConsole({
```

Comment:
I think it'd be good to have some output even in production. Maybe we could introduce a command line arg for switching that.